.github/scripts/process_commit.py

```diff
@@ -41,26 +42,32 @@ def get_pr_number(commit_hash: str) -> Any:
     return data[0]["number"]
 
 
-def get_pr_merger_and_labels(pr_number: int) -> Tuple[str, Set[str]]:
+def get_pr_user_and_labels(pr_number: int) -> Tuple[str, Set[str]]:
```

Comment:
```suggestion
def get_pr_author_and_labels(pr_number: int) -> Tuple[str, Set[str]]:
```

Reply:
This terminology comes from Github API. Reference:  https://docs.github.com/en/rest/reference/pulls#list-pull-requests--code-samples . Hence kept here as well for consistency. But true we are actually looking for authon.